### DISCUSSION TITLE
hi, where si changelog and why are not new versions tracked in github?

### DISCUSSION THREAD

**Discussion by tomasstrejcek** [Urigo/graphql-scalars#1521] [Q&A]
title ^

**Comment by serhalp** [comment]
@ardatan Yes, the last 18 releases are not documented anywhere. Is it possible to please consistently document release notes? I'd recommend using tooling to automate this so it's easy and impossible to forget, such as https://www.npmjs.com/package/conventional-github-releaser.

Thank you!
